sql/upgrade/sl3.0.sql

```diff
@@ -376,6 +373,22 @@ $$
         SELECT entity_id from users where username = SESSION_USER OR username = 'Migrator';
 $$ LANGUAGE SQL;
 
+-- Make sure transactions is filled. The actual trigger will be redefined
+-- in modules later.
+
+CREATE OR REPLACE FUNCTION track_global_sequence()
```

Comment:
Why do you need this? We can insert the records in the transactions table based on what's in the gl/ar/ap tables right after filling the ar/ap/gl tables, right? Or we can fill the transactions table in the same query as we're using to fill the ar/ap/gl tables?

Reply:
Transactions table in 1.6 didn't have all the fields it now possesses, so we cannot rely on the current trigger to fill the transaction table.

Reply:
Right. I guess my point was that after INSERT INTO ar SELECT ... FROM :slschema.ar, you could: INSERT INTO transactions (id, ...) SELECT id, ... FROM ar; then you don't need the trigger.

Come to think of it, I think the insertions need to happen into the transactions table before the insertions into the ar/ap/gl tables.